pkg/controller/operators/catalog/operator.go

```diff
@@ -186,7 +186,8 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 		bundleUnpackTimeout:      bundleUnpackTimeout,
 		clientFactory:            clients.NewFactory(config),
 	}
-	op.sources = grpc.NewSourceStore(logger, 10*time.Second, 10*time.Minute, op.syncSourceState)
+
+	op.sources = grpc.NewSourceStore(logger, 10*time.Second, catalogSourceReadyTimeout, op.syncSourceState)
```

Comment:
The `readyTimeout` parameter controls the amount of time we wait for a state change from the gRPC "ready" state. It only exists to keep us from blocking forever [here][1], and doesn't affect how quickly (or slowly) operator upgrades occur/are reconciled.

[1]: https://github.com/operator-framework/operator-lifecycle-manager/blob/c440dc1ae1334696f1c1de051d6a631b097857a4/pkg/controller/registry/grpc/source.go#L214